components/list/test/list-item-drag-handle.visual-diff.js

```diff
@@ -0,0 +1,48 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+
+describe('d2l-list-item-drag-handle', () => {
+
+	const visualDiff = new VisualDiff('list-item-drag-handle', __dirname);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await visualDiff.createPage(browser);
+		await page.goto(`${visualDiff.getBaseUrl()}/components/list/test/list-item-drag-handle.visual-diff.html`, {waitUntil: ['networkidle0', 'load']});
+		await page.bringToFront();
+	});
+
+	after(async() => await browser.close());
+
+	beforeEach(async() => {
```

Comment:
Nit: in the interest of brevity and matching the line directly above it, this could be:
```
beforeEach(async() => await visualDiff.resetFocus(page));
```
No big deal though.